gravitee-apim-console-webui/src/organization/configuration/tags/org-settings-tags.component.ts

```diff
@@ -25,16 +38,47 @@ import { Subject } from 'rxjs';
 export class OrgSettingsTagsComponent implements OnInit, OnDestroy {
   isLoading = true;
 
+  providedConfigurationMessage = 'Configuration provided by the system';
+
+  tagsTableDS: TagTableDS[];
```

Comment:
Same here as we are not using MatTableDataSource
```suggestion
  tags: TagVM[];
```